testing/tests/DevExpress.exporter/jspdfParts/jspdf_v2.dataGrid.styles.tests.js

```diff
@@ -0,0 +1,375 @@
+import { exportDataGrid } from 'exporter/jspdf/export_data_grid_2';
+
+const JSPdfStylesTests = {
+    runTests(moduleConfig, createMockPdfDoc, createDataGrid) {
+
+        const onRowExporting = (e) => { e.rowHeight = 16; };
+
+        const rowTypeStyles = {
+            headerStyles: { backgroundColor: '#808080' },
+            groupStyles: { backgroundColor: '#d3d3d3' },
+            totalStyles: { backgroundColor: '#ffffe0' }
+        };
+
+        QUnit.module('Styles', moduleConfig, () => {
+            QUnit.test('Simple - [{f1, f2]', function(assert) {
+                const done = assert.async();
+                const doc = createMockPdfDoc();
+
+                const dataGrid = createDataGrid({
+                    columns: [
+                        { dataField: 'f1' },
+                        { dataField: 'f2' },
+                    ],
+                    dataSource: [
+                        { f1: 'f1_1', f2: 'f1_2' },
+                    ],
+                });
+
+                const expectedLog = [
+                    'setFillColor,#808080', 'rect,10,15,90,16,F',
+                    'text,F1,10,23,{baseline:middle}', 'setLineWidth,1', 'rect,10,15,90,16',
+                    'setFillColor,#808080', 'rect,100,15,80,16,F',
+                    'text,F2,100,23,{baseline:middle}', 'setLineWidth,1', 'rect,100,15,80,16',
+                    'text,f1_1,10,39,{baseline:middle}', 'setLineWidth,1', 'rect,10,31,90,16',
+                    'text,f1_2,100,39,{baseline:middle}', 'setLineWidth,1', 'rect,100,31,80,16'
+                ];
+
+                exportDataGrid(doc, dataGrid, { topLeft: { x: 10, y: 15 }, columnWidths: [ 90, 80 ], onRowExporting, ...rowTypeStyles }).then(() => {
```

Comment:
Styles were constructed in an object and destructed here.
Internally, styles will be constructed back to a single object.
Can we pass an object without destruct/construct?